### ISSUE TITLE
There is a small documentation error here. 

### ISSUE THREAD

**Issue by RobertFloor** [ansible-middleware/amq#40]
##### SUMMARY
There is a small documentation error here: https://github.com/ansible-middleware/amq/tree/main/roles/activemq

##### ISSUE TYPE
 - Bug Report

##### ACTUAL RESULTS
It should be activemq_tls_keystore_pa**s**sword instead of activemq_tls_keystore_pasword .  
```
Sample connector with TLS:

  - name: amqp
    address: 172.168.10.43
    port: 61616
    parameters:
      tcpSendBufferSize: 1048576
      tcpReceiveBufferSize: 1048576
      protocols: CORE
      useEpoll: true
      sslEnabled: True
      keyStorePath: "{{ activemq_tls_keystore_dest }}"
      keyStorePassword: "{{ activemq_tls_keystore_pasword }}"
      trustStorePath: "{{ activemq_tls_truststore_dest }}"
      trustStorePassword: "{{ activemq_tls_truststore_password }}"
      verifyHost: False
```

**Comment by guidograzioli** [comment]
great catch!

**Comment by RobertFloor** [comment]
:D Found it thanks to a spellchecker in my IDE: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

**Comment by guidograzioli** [comment]
Fixed by #41
